packages/cli/src/utils/build.js

```diff
@@ -64,11 +64,12 @@ const findRequiredFiles = async (workingDir, entryPoints) => {
     platform: 'node',
     metafile: true,
     logLevel: 'warning',
+    logOverride: {
+      'require-resolve-not-external': 'silent',
+    },
     external: [
       '../test/userapp',
       'zapier-platform-core/src/http-middlewares/before/sanatize-headers', // appears in zapier-platform-legacy-scripting-runner/index.js
-      './request-worker', // appears in zapier-platform-legacy-scripting-runner/zfactory.js
-      './xhr-sync-worker.js', // appears in jsdom/living/xmlhttprequest.js
```

Comment:
This is actually external - I mean from 3rd-party package. I think we can leave this.